packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -4,6 +4,19 @@
 
 part of google_maps_flutter;
 
+const MethodChannel _channel =
+    MethodChannel('flutter.io/googleMapsPluginUtils');
+
+/// Utils for GoogleMaps plugin
+class GoogleMapsPluginUtils {
+  ///This method creates dummy map. This call will initialize all services needed by GoogleMaps This
+  ///will speed up next GoogleMap view initialization. Android only.
+  static Future<void> warmUp() {
+    if (!Platform.isAndroid) throw Exception("warmUp is Android only");
+    return _channel.invokeMethod<void>('warmUp');
+  }
```

Comment:
This should become a method in the `platform_interface` of the map, [like this one here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart#L235-L240) (and its [implementation](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart#L432-L438)).